app/screens/sso/sso.js

```diff
@@ -119,7 +119,13 @@ class SSO extends PureComponent {
 
         CookieManager.get(parsedUrl.href, true).then((res) => {
             const mmtoken = res.MMAUTHTOKEN;
+            const csrf = res.MMCSRF;
             const token = typeof mmtoken === 'object' ? mmtoken.value : mmtoken;
+            const csfrToken = typeof csrf === 'object' ? csrf.value : csrf;
```

Comment:
This variable seems to have a typo in it.

Reply:
fixed!